README.md

```diff
@@ -1,11 +1,31 @@
-
 # encoding-down
 
-WIP LevelDOWN wrapper supporting levelup@1 encodings. For motivation, see [this issue](https://github.com/Level/levelup/pull/367).
+[`AbstractLevelDOWN`](https://github.com/level/abstract-leveldown) wrapper supporting levelup@1 encodings. For motivation, see [this issue](https://github.com/Level/levelup/pull/367).
+
+## Usage
+
+```js
+var levelup = require('levelup')
+var leveldown = require('leveldown')
+var enc = require('encoding-down')
+const db = levelup(enc(leveldown('./db')))
+db.put('name', 'encoding-down', function (err) {
+  db.get('name', function (err, value) {
+    if (!err) console.log('name=', value)
+  })
+})
```

Comment:
Aah my bad. First I wrote only const then replaced with var. Wdyt? I don't really care which as long as it's consistent